src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java

```diff
@@ -49,24 +51,30 @@ public void should_complete_envelope_referenced_by_queue_message() throws Except
         String zipFilename = uploadEnvelope();
 
         await("Envelope should be created in the service and notification should be put on the queue")
-            .atMost(scanDelay + MAX_MESSAGE_PROCESSING_TIME_MILLIS, TimeUnit.MILLISECONDS)
+            .atMost(scanDelay + MESSAGE_PROCESSING_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS)
             .pollInterval(500, TimeUnit.MILLISECONDS)
             .until(() -> hasNotificationBeenSent(zipFilename));
 
+        assertThat(testHelper.storageHasFile(inputContainer, zipFilename)).isTrue();
         UUID envelopeId = getEnvelope(zipFilename).getId();
 
         // when
         sendProcessedEnvelopeMessage(envelopeId);
 
         // then
         await("Envelope should change status to 'COMPLETED'")
-            .atMost(MAX_ENVELOPE_FINALISATION_TIME_MILLIS, TimeUnit.MILLISECONDS)
+            .atMost(ENVELOPE_FINALISATION_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS)
             .pollInterval(500, TimeUnit.MILLISECONDS)
             .until(() -> getEnvelope(zipFilename).getStatus() == Status.COMPLETED);
 
         EnvelopeResponse updatedEnvelope = getEnvelope(zipFilename);
         assertThat(updatedEnvelope.getScannableItems()).hasSize(2);
         assertThat(updatedEnvelope.getScannableItems()).allMatch(item -> item.ocrData == null);
+
+        await("file should be deleted")
+            .atMost(DELETE_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS)
+            .pollInterval(2, TimeUnit.SECONDS)
+            .until(() -> testHelper.storageHasFile(inputContainer, zipFilename), is(false));
     }
```

Comment:
Sorry, I didn't notice it before, but we've remove all the assertions for `Envelope.isZipDeleted()`. We need to put it here.

Reply:
Sorry, not here, but in an appropriate integration test.